src/main/webapp/app/exercises/programming/shared/code-editor/instructions/code-editor-instructions.component.html

```diff
@@ -1,12 +1,19 @@
-<div id="cardInstructions" class="instructions-wrapper card border border-lightgrey">
-    <div class="card-header d-flex justify-content-center text-white bg-primary" (click)="toggleEditorCollapse($event)">
+<div id="cardInstructions" class="instructions-wrapper card collapsable-card" [ngStyle]="collapsed ? { height: '250px' } : { height: '100%' }">
```

Comment:
I know it's just one style option in the ngStyle clause but it's still inline css which is a bad practice. You can consider moving the height options to classes.